code/game/objects/structures/safe.dm

```diff
@@ -23,9 +27,20 @@ FLOOR SAFES
 	var/space = 0		//the combined w_class of everything in the safe
 	var/maxspace = 24	//the maximum combined w_class of stuff in the safe
 	var/combo_to_open	//so admins know the code
+	var/obj/item/thermal_drill/drill = null
+	var/drill_timer
+	var/time_to_drill
+	var/image/bar
+	var/drill_start_time
+	var/datum/looping_sound/thermal_drill/soundloop
+	var/datum/effect_system/spark_spread/spark_system

```

Comment:
I don't think it's necessary to have a holder var for this as it won't be a regularly used thing.